backend/app/Repositories/EloquentWebsiteRepository.php

```diff
@@ -23,4 +23,14 @@ public function getById(int $id): Website
     {
         return Website::findOrFail($id);
     }
+
+    public function getCurrentWebsite(): ?Website
+    {
+        try {
+            $websiteId = auth()->user()->website->id;
```

Comment:
It is incorrect, as from tracking website will not be done any authorized requests. It is impossible